source/includes/_endpoint-deck.md

```diff
@@ -280,9 +280,11 @@ display | object | Stores settings used to load the analysis
 
 #### POST
 
-To create a new slide, POST an analysis to the slides catalog. The payload, described in the [below](#analyses) section, should be wrapped as a shoji:entity.
+To create a new slide, POST an analysis to the slides catalog. The payload,
+described in the [below](#analyses) section, should be wrapped as a shoji:entity.
 
-On success, the server returns a 201 response with a Location header containing the URL of the newly created slide entity.
+On success, the server returns a 201 response with a Location header containing
+the URL of the newly created slide entity with its first analysis.
```

Comment:
It only allows to create the slide with the first analysis, For subsequent analyses you need to POST to that slide's analyses catalog. Adding to the docs.